internal/utils/common.go

```diff
@@ -348,3 +350,31 @@ func Copy(src, dst string) error {
 	}
 	return nil
 }
+
+// GetSecureboot checks if secureboot is enabled in the system by checking the expected efivar
+func GetSecureboot() bool {
```

Comment:
Manually checking the Efivar for Secureboot support saves us from depending on go-uefi. we only used this part and checking a single efi var is simple